src/components/ProfileForm/components/ProfileForm/ProfileForm.css

```diff
@@ -0,0 +1,6 @@
+progress::-webkit-progress-bar {
+    background-color: white;
+ }
+ progress::-webkit-progress-value {
+    background-color: black;
+ }
```

Comment:
We should add some classes. Otherwise this rule will apply to ALL progress bars in the app. Are we sure we want that?